app/carbonapi/http_handlers_test.go

```diff
@@ -16,9 +16,8 @@ func TestShouldNotBlock(t *testing.T) {
 
 	req.Header.Add("foo", "bar")
 	rule := Rule{"foo": "block"}
-	config.blockHeaderRules = RuleConfig{Rules: []Rule{rule}}
 
-	if shouldBlockRequest(req) {
+	if shouldBlockRequest(req, []Rule{rule}) {
```

Comment:
This looks like a pre-existing condition, but I think either the func name or the error message are wrong: "shouldBlockRequest" returns true, we error saying "should not have blocked request"?